core/scm.js

```diff
@@ -187,7 +187,24 @@ const SCHEMA_HOOK = Joi.object().keys({
         .required()
         .label('Type of the event'),
 
-    username: Joi.reach(SCHEMA_USER, 'username')
+    username: Joi.reach(SCHEMA_USER, 'username'),
+
+    releaseId: Joi.number()
+        .integer()
```

Comment:
We should use `string`. Maybe another SCM's release IDs include non-numbers.

Reply:
Thank you your comments. I changed type of releaseId number to string.